public-ui/src/app/data-browser/views/ehr-view/ehr-view.component.html

```diff
@@ -24,9 +24,23 @@ <h5 id="domain-summary" class="secondary-display">
                   <clr-icon size="20" *ngIf="showTopConcepts" shape="caret" dir="down"></clr-icon>
                 </div>
               </h5>
-              <div *ngIf="showTopConcepts">
+              <div *ngIf="showTopConcepts && !displayConceptErrorMessage">
                 <chart-top-results (topResultSelected)="selectConcept($event,true)" [concepts]="top10Results"></chart-top-results>
               </div>
+              <div *ngIf="showTopConcepts && displayConceptErrorMessage" class="col col-xs-7 error-message">
+                  <div class="alert alert-box">
+                    <div class="alert-items">
+                      <div class="alert-item static">
+                        <div class="alert-icon-wrapper">
+                          <clr-icon class="alert-icon" shape="exclamation-triangle" size="26"></clr-icon>
+                        </div>
+                        <span class="alert-text">
+          Sorry, the data are currently unavailable. Please try refreshing the page or returning home <a [routerLink]="['']">here.</a>
+        </span>
+                      </div>
+                    </div>
+                  </div>
```

Comment:
This code seems to be repeated several times. Doesn't necessarily need to be included in this PR but might help to create a reusable error message component. 

Optional nit: Looks like you're using Clarity's alert system. I've found it fairly simple create our own messaging and eliminate all the nested divs.
(Sorry, the examples below are in React but the styles should work in Angular also)

[Error element](https://github.com/all-of-us/workbench/blob/808175c2c27cf0f4e56eeed2a7095fc87ef9e760/ui/src/app/pages/data/concept/concept-homepage.tsx#L465)
```
errorMessage() {
  return <div style={styles.error}>
    <ClrIcon style={{margin: '0 0.5rem 0 0.25rem'}} className='is-solid' shape='exclamation-triangle' size='22'/>
    Sorry, the request cannot be completed. Please try refreshing the page or contact Support in the left hand navigation.
  </div>;
}
```

[Error style](https://github.com/all-of-us/workbench/blob/808175c2c27cf0f4e56eeed2a7095fc87ef9e760/ui/src/app/pages/data/concept/concept-homepage.tsx#L85)
```
error: {
    background: colors.warning,
    color: colors.white,
    fontSize: '12px',
    fontWeight: 500,
    textAlign: 'left',
    border: '1px solid #ebafa6',
    borderRadius: '5px',
    marginTop: '0.25rem',
    padding: '8px',
}
```

Reply:
Split this into a seperate component.